tests/generation/test_utils.py

```diff
@@ -1520,66 +1525,53 @@ def test_assisted_decoding_matches_greedy_search(self):
                 self.skipTest("Won't fix: old model with different cache format")
             if any(
                 model_name in model_class.__name__.lower()
-                for model_name in ["bigbirdpegasus", "led", "mega", "speech2text", "git", "prophetnet"]
+                for model_name in ["bigbirdpegasus", "led", "mega", "speech2text", "git", "prophetnet", "seamlessm4t"]
```

Comment:
Note: `seamlessm4t` was already in the skip list of `test_assisted_decoding_sample`, probably for the same post mortem reasons